### DISCUSSION TITLE
Two people cannot work in the same project

### DISCUSSION THREAD

**Discussion by erakastrati** [Budibase/budibase#2413] [Help] [answered]
Hi there,

Why can't two users work at the same time in the same project?
As soon as I join the project, my colleauge’s project is locked.

**Comment by kevmodrome** [comment, marked as answer]
This is a technical limitation of the platform at the moment. It's something we've discussed internally and will most likely a feature that will be showing up down the road (not in the near future).

**Reply by erakastrati** [reply]
Thanks a lot for the answer @kevmodrome

**Comment by hsum** [comment]
Is the technical limitation related to the backend storage?  I'm using a postgresql server, which wouldn't limit multiple users from writing.

Allowing multiple users would open up many possibilities for budibase to nicely integrate into existing database applications.  I'm sure I'm not alone in wanting to replace of some legacy code with a polished solution.

It seems like you can use the same account and log in multiple times.  Any drawbacks to this workaround?  Any more on the nature of the technical limitation?

thank you!
